docs/proposals/PROPOSAL_H.md

```diff
@@ -76,11 +78,11 @@ Schema:
 
     - **`compatibility feature field`**
 
-      This REQUIRED property describes a feature compatibility requirement on the target host, the field is named with NFD feature label.
+      This REQUIRED property describes a feature compatibility requirement on the target host, the field is named with OCI feature label.
```

Comment:
Changing this from an NFD label to an OCI label I believe means that OCI would be maintaining a parallel spec to NFD and others in this space. That adds the risk that the OCI spec will get out of sync with features of the others, creating complication for users trying to support them all.

Reply:
OK, to reduce the risk of out of sync, NFD labels could be reused. OCI and NFD may need to collaborate to review all labels, to meet both demands.

Reply:
I would change:
> This REQUIRED property describes a feature compatibility requirement on the target host, the field is named with OCI feature label.

to

> This REQUIRED property describes a compatibility requirement on the target host, the field accepts OCI and community defined labels.

Reply:
good idea